tests/scipy_signal_test.py

```diff
@@ -106,5 +106,219 @@ def testDetrend(self, shape, dtype, axis, type, bp):
     self._CompileAndCheck(jsp_fun, args_maker, rtol=tol, atol=tol)
 
 
+_stft_test_shapes = [
+    # (input_shape, nperseg, noverlap, axis)
+    ((50,), 17, 5, -1),
+    ((2, 13), 7, 0, -1),
+    ((3, 17, 2), 9, 3, 1),
+    ((2, 3, 389, 5), 17, 13, 2),
+]
+
+
+@jtu.with_config(jax_numpy_rank_promotion="raise")
```

Comment:
You can leave this out, as it will be set by default once #9330 is merged. Same below.

Reply:
those test classes were merged into `LaxBackedScipySignalTests`.
So, done for `LaxBackedScipySignalTests`